pkg/files.go

```diff
@@ -0,0 +1,57 @@
+package files
```

Comment:
Move this under `internal/store/download.go` for now. We'll move it under pkg later.